contracts/DCAHub/DCAHubSwapHandler.sol

```diff
@@ -150,31 +166,26 @@ abstract contract DCAHubSwapHandler is ReentrancyGuard, DCAHubConfigHandler, IDC
       _pairInSwap.tokenA = _tokens[indexTokenA];
       _pairInSwap.tokenB = _tokens[indexTokenB];
 
-      uint256 _amountToSwapTokenA;
-      uint256 _amountToSwapTokenB;
-
-      (_amountToSwapTokenA, _amountToSwapTokenB, _pairInSwap.intervalsInSwap) = _getTotalAmountsToSwap(
+      (_pairInSwap.totalAmountToSwapTokenA, _pairInSwap.totalAmountToSwapTokenB, _pairInSwap.intervalsInSwap) = _getTotalAmountsToSwap(
```

Comment:
Will test that `totalAmountToSwapTokenA` and `totalAmountToSwapTokenB` are expose correctly in another PR